src/omotes_sdk/internal/common/broker_interface.py

```diff
@@ -113,6 +115,51 @@ def to_argument(self) -> AioPikaQueueTypeArguments:
         return result
 
 
+@dataclass()
+class QueueMessageTTLArguments():
+    """Construct additional time-to-live arguments when declaring a queue."""
+
+    queue_ttl: Optional[timedelta] = None
+    """Expires and deletes the queue after a period of inactivity.
+    The timedelta must be able to be casted into a positive integer."""
+    message_ttl: Optional[timedelta] = None
+    """Expires and deletes the message within the queue after the defined TTL.
+    The timedelta must be able to be casted into a non-negative integer."""
+    dead_letter_routing_key: Optional[str] = None
+    """When specified, the expired message is republished to the designated dead letter queue.
+    If not set, the message's own routing key is used."""
+    dead_letter_exchange: Optional[str] = None
+    """Dead letter exchange name."""
+
+    def to_argument(self) -> Arguments:
```

Comment:
Perhaps this function is a good candidate for unit testing?

Reply:
Added unit tests in 3691c6257cb2daa7130fb39e33bb8f64de43e7ab